pkg/apis/autoscaling/v1alpha1/pa_types.go

```diff
@@ -114,6 +114,8 @@ const (
 	PodAutoscalerConditionReady = apis.ConditionReady
 	// PodAutoscalerConditionActive is set when the PodAutoscaler's ScaleTargetRef is receiving traffic.
 	PodAutoscalerConditionActive apis.ConditionType = "Active"
+	// PodAutoscalerConditionInitiallyActive is set when PodAutoscaler has first reached its initial scale.
+	PodAutoscalerConditionInitiallyActive apis.ConditionType = "InitiallyActive"
```

Comment:
Is this an informational status? If so we should comment to that. If not — it'd be interesting to know why.
Also should it be a status, or perhaps this is a good thing to put in the `status.Annotations`?

Reply:
My 2c is this is informational. As a user the behaviour will be different before/after first activation, so knowing if the revision successfully passed its first activation is a reasonable thing for a user to want to see in conditions.

Reply:
I don't think this is informational for the users. We can put this into `status.Annotations`.

Reply:
(not that I’m super bothered personally, an annotations works too)

Reply:
I guess we can always expose it as a condition later if we want to 🤷‍♂️

Reply:
To be honest, I think the other way round is actually right. Today our PA becomes non-ready if the revision is scaled to zero, which is... weird and causes race issues with revision readiness.

Reply:
@markusthoemmes are you suggesting changing the semantics of the existing condition or adding a vote for creating this new one?

Reply:
That's a good question, I think I was mostly rumbling as these changes are kinda out of scope. Maybe it makes sense to think through them before adding new conditions though.

Reply:
SO what's the consensus? Informational status for now and long overdue refactoring later?

Reply:
I think on slack we landed on a true/false status for now, so long as you agree too :)

Reply:
Sure, but true/false can be mandatory (i.e. included in happy computation) or informational :)

Reply:
Eh, status.Annotation :) Sure.